scripts/generate-release-notes.sh

```diff
@@ -143,6 +143,23 @@ sha1sum of genesis wasm   : $GENESIS_WASM_HASH
 EOF
 fi
 
+# release notes for enclave binary 
+if [ "$2" != "runtime" ] && [ "$2" != "client" ] && [ "$2" != "both"]; then 
+   echo "Generating Release Notes for Enclave"
+   local UPSTREAM_COMMIT=$(cat tee-worker/upstream_commit)
```

Comment:
why need `UPSTREAM_COMMIT` here? Or in another words: Parachain repo commit hash is less important?

Reply:
I wasn't sure, what all needs to be included in the `release notes` but I thought `UPSTREAM_COMMIT` could be good to track the progress of our worker against the upstream when we release a version from our side.

Reply:
I don't think `UPSTREAM_COMMIT` should be included in the release note. It's mainly used for internal usage to track against the upstream worker. I would suggest the current repo git commit hash is more important and valuable. Take the short hash value.

Reply:
Removed ✅